### PR TITLE
Sleeping Carp Buff

### PR DESCRIPTION
**Pull request by ghost**
- The bo staff now has a weight class of 3, allowing it to fit in backpacks.
- A carphide vest may now be purchased for 20 Influence (subject to change). This vest provides complete protection from electrodes and high protection from bullets and blunt weaponry across the entire body excluding the head. Anyone not in the Sleeping Carp gang will take continuous damage if they attempt to wear the vest. This vest replaces the Wrestling Belt in the gangtool.

### CODE REVIEW

File: code/datums/martial.dm [hunk 1]
```diff
@@ -554,3 +554,32 @@
 		return 1
 	else
 		return 0
+
+/obj/item/clothing/suit/sleeping_carp
+	name = "carphide vest"
+	desc = "A strange armor vest formed of toughened space carp hide. Excellent at deflecting energy blasts. <span class='warning'>The collar is lined with a circlet of razor-sharp teeth.</span>"
+	icon_state = "carpvest"
+	item_state = "armor"
+	armor = list(melee = 60, bullet = 20, laser = 100, energy = 100, bomb = 5, bio = 0, rad = 0) //Complete protection from energy blasts
```
Comment (Incoming5643) [inline code/datums/martial.dm]: min(armor,90) means that giving an armor 100% armoring isn't practical. You could always cheat and give it deflection mechanics though.

**Comment by SconesC** [comment]
Why

That armor is so obscenely strong, stacking that with a gang uniform is just dumb. Neutering Security's ability to lethal these guys is... I thought the big complaint was that they were losing fights with other gangs? Who have guns?

I can't get over how fucking strong that armor vest is

**Comment by Dorsisdwarf** [comment]
:-1: 

Leave security aloooooooooooooooooone

**Comment by WJohn** [comment]
That armor is much, much, MUCH stronger than anything even the deathsquad have. Holy shit dude.

**Comment by Xhuis** [comment]
Armor will be getting significant overhauls shortly - remember that this is very early and _everything_ is subject to change!

**Comment by Xhuis** [comment]
Armor is nerfed and now functions primarily as a ballistic vest.

**Comment by Xhuis** [comment]
This is merge-ready unless someone thinks another change should be made or something else added.

**Comment by SconesC** [comment]
Energy = 100

Did you even read what I wrote?

**Comment by AlexanderUlanH** [comment]
It's not really, really stupid because the carp gang has no ranged weaponry, making them extremely weak against security, especially when trying to defend a dominator.  They're still vulnerable to stun batons, just like dual esworders who also are immune to tasers and get dunked regularly.

**Comment by optimumtact** [comment]
It's the ranged stun that really fucks them over

**Comment by Xhuis** [comment]
The DNM tag can be removed from this now

@SconesC I gave them full taser resistance because with no ranged weaponry they get utterly crushed by anyone carrying a taser.

**Comment by Rinoka** [comment]
Maybe make it so that they recover from stun exponentially faster/get an implant to immediately recover from stun/etc? Or make it so that they can block all taser shots if they have something that resists electrical shocks on their hands, via KUNG FU BULLET CATCHING MAGIC?

I'm cool with them getting a buff to their abilities, but maybe just giving them more powerful things that fit their style (IE, defensive hand stance + metallic gloves for bullets/electrical gloves for tasers letting you catch bullets/taser shots and nullify them) would be more fitting than hurling a fairly powerful vest at them that only they can use.

**Comment by Xhuis** [comment]
I wanted to give them a vest so they couldn't combo unique projectile dodging/catching with really good armor and become invincible to literally everything. A vest fits the style (super sekrit ninja sect armor) of the gang I was aiming for but also stays relatively balanced.

**Comment by SconesC** [comment]
I've actually seen sleeping carp win a lot recently

>brain damage death stick

**Comment by Xhuis** [comment]
Kappa

**Comment by Xhuis** [comment]
Any reason this can't be merged?

**Comment by SconesC** [comment]
Several

**Comment by Xhuis** [comment]
Please voice your concerns and I'll remedy them.

**Comment by Boredone** [comment]
So, don't know much, but two questions.
1: What happens if you shoot them in the head with a taser?
2: Isn't there a thing capping total protection at 90% or so to prevent you from becoming literally invincible to things since people stacked stuff to become immune to melee damage?

**Comment by SconesC** [comment]
>melee = 60, bullet = 80, laser = 25, energy = 100

These are still better than DS and combined with a gang uniform they're basically fucking invincible

**Comment by Xhuis** [comment]
@Boredone:
1. No, they are stunned as normal.
2. Not that I'm aware of.

@SconesC:
Their armor is extremely powerful because they lack ranged capabilities. Anyone with a taser can easily kill even someone with a gang uniform and martial arts. Being melee only, they need powerful protection against ranged in order to stand up at all against the other gangs.

**Comment by SconesC** [comment]
I'm not asking you to justify it

I'm telling you it's awful. It's not up for debate, those values regardless of context are unacceptable.

The martial arts are always going to be niche but good. If you really insist on giving them resists, make it active instead of passive armor by way of giving the bo staff some decent (60%) block chance for bullets and energy projectiles

**Comment by Boredone** [comment]
So, here's some apparently obvious problems to me with the facts.
1. Officers and other Gangers can just shoot the head, ignoring all the protection, making them vulnerable to ranged still.
2. If officers don't do that, what happens when they get robusted and have their tasers stolen? You have a ganger immune to tasers shooting tasers at people then beating their heads in.

**Comment by Xhuis** [comment]
I can understand your concerns and I will be doing my best to change the PR when I get the time

**Comment by SconesC** [comment]
Giving the staff block is honestly a pretty decent idea

**Comment by Xhuis** [comment]
I don't have the time to fix the conflicts on this, and it looks like it's going nowhere anyway. If I feel this is a necessary re-open then I will do it, but I've been told that Sleeping Carp winrates have been increasing and people are learning how to play the gang properly.

**Pull request closed without merging**